Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/SearchResultHelper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Flowpack\ElasticSearch\ContentRepositoryAdaptor\Eel;
+
+/*
+ * This file is part of the Flowpack.ElasticSearch.ContentRepositoryAdaptor package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+use TYPO3\Eel\ProtectedContextAwareInterface;
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * Eel Helper to process the ElasticSearch Query Result
+ */
+class SearchResultHelper implements ProtectedContextAwareInterface
+{
+
+    /**
+     * @param ElasticSearchQueryResult $searchResult
```

Comment:
Please add a nice description. Imagine we'd be generating Eel helper references from this…